### PR TITLE
[MGW-238] Tweak documentation to clarify service catalog limitation

### PR DESCRIPTION
**Pull request by leobellaera**
[Jira Ticket](https://www.mulesoft.org/jira/browse/MGW-238)

### CODE REVIEW

**Review by dosten — DISMISSED** [review]
LGTM, let's wait for Priya's approval

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 1]
```diff
@@ -15,16 +15,16 @@ Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you mu
 [[supported_versions]]
 == Software Requirements
 
-Your Anypoint Service Mesh installation requires the following applications and Kubernetes environments:
+Your Anypoint Service Mesh installation requires you to:
 
-* One of the following environments on which to run Kubernetes:
+* Have installed one of the following environments on which to run Kubernetes:
```
Comment (dukesphere) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: we usually avoid tenses like "have install"

The prerequisite are:
* Install x

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc [hunk 2]
```diff
@@ -15,16 +15,16 @@ Anypoint Service Mesh is developed using Istio and Kubernetes. Therefore, you mu
...
 ** Google Kubernetes Engine (GKE)
 ** Amazon EKS
 ** Azure Kubernetes Service (AKS)
 ** Red Hat OpenShift
-* Kubernetes (versions 1.12 through 1.20) or Red Hat OpenShift (version 4.x)
-
-* <<install_Istio,Istio>> (currently supported versions include 1.7.x, 1.8.x, 1.9.x, and 1.10.x)
+* Have installed Kubernetes (versions 1.12 through 1.20) or Red Hat OpenShift (version 4.x)
+* Have installedd <<install_Istio,Istio>> (currently supported versions include 1.7.x, 1.8.x, 1.9.x, and 1.10.x)
+* Not have installed https://svc-cat.io/[Service Catalog^] in the target cluster
```
Comment (dukesphere) [inline modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Prob say separately "Do not install" 

And prob should tell them what to do if they have already installed the Service Catalog. Uninstall? Anything else?

**Review by dukesphere — DISMISSED** [review]
some style and substantive suggestions

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

File: modules/ROOT/pages/prepare-to-install-service-mesh.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/prepare-to-install-service-mesh.adoc]: Done! Thanks!

**Review by dukesphere — APPROVED** [review]

**Comment by leobellaera** [comment]
@dosten Do you think it's ok?

**Comment by dosten** [comment]
@leobellaera @dukesphere it's ok from my side

**Pull request merged**